dcc-submission-server/src/main/java/org/icgc/dcc/submission/server/sftp/SftpServerService.java

```diff
@@ -168,15 +184,25 @@ private String getLogMessage(String username) {
   /**
    * Returns some of the useful values for an {@link IoSession}.
    */
-  private Map<String, String> getIoSessionMap(IoSession ioSession) {
+  private Map<String, String> getIoSessionMap(AbstractSession session) {
+    val ioSession = session.getIoSession();
     val map = Maps.<String, String> newLinkedHashMap();
     map.put("id", valueOf(ioSession.getId()));
     map.put("localAddress", ioSession.getLocalAddress().toString());
     map.put("remoteAddress", ioSession.getRemoteAddress().toString());
 
+    val transfer = session.getAttribute(FILE_TRANSFER_SESSION_ATTRIBUTE);
+    if (hasFileTransfer(transfer)) {
+      map.put("fileTransfer", transfer.getPath());
```

Comment:
Couldn't there be more than one? For example an `MPUT`. Not sure if this is in parallel or not.